Quantum-Control-Applications/NV-Centers/Confocal-Setup/cw_odmr.py

```diff
@@ -28,7 +29,7 @@
         with for_(f, f_min, f <= f_max, f + df):  # Notice it's <= to include f_max (This is only for integers!)
             update_frequency("NV", f)  # update frequency
             align()  # align all elements
-
+            wait(20, "NV", "AOM")  # Used to synchronize AOM and SPCM (from simulation)
```

Comment:
And delay has to be calibrated and added to the configuration, AOMs have delays that can be 400ns long... so getting the pulses to be the same in the simulator doesn't really help.
I suggest removing it

Reply:
Yes I removed it